### ISSUE TITLE
White header color should default a different icon color

### ISSUE THREAD

**Issue by kevindherman** [proudcity/wp-proudcity#1539]
https://www.warracres-ok.gov/

Can white headers default icons to black. The icons on warr acres is showing as white so you can't see them with the latest update.

**Comment by jlyon** [comment]
@kevindherman I made the temp fix we talked about last week by adding custom css. Closing this for now, but let us know if there are any other reports of weirdness
```
a.card-btn fa {
	color: black;
}
```

**Comment by kevindherman** [comment]
@lukefretwell brought up these points, perhaps we should merge the issues: https://github.com/proudcity/wp-proudcity/issues/1541

@jlyon
